### DISCUSSION TITLE
4-key osu! standard

### DISCUSSION THREAD

**Discussion by Dip9107** [ppy/osu#19860] [Ideas]
With the recent ranking of many high star rating maps, I feel like it would push osu! leaps and bounds forward if a 4-key osu! standard mode was added to lazer with separate ranking and mapset leaderboards (just like how osu! standard, mania, taiko, and CTB are all their own separate selectable game modes from the song select screen) as to differ between 2-key playstyle and 4-key playstyle. Another way it could be implemented is as a mod instead of its own separate game mode, but I feel as if the community would take it better if it were it’s own mode so that 2-key scores wouldn’t all be erased from leaderboards with 4-key scores if it gains traction and players take the time to learn it. I know osu! is a 2-key game and always has been, but with how much the game has moved in terms of skill since it’s creation I feel as if it’s getting to a point where players are reaching a maximum possible BPM that is able to be humanly tapped with only 2 input buttons at their disposal. Creating a 4-key mode would make so much more possible for players with the skill and motivation to learn it and possibly even give some of these new extremely hard maps some more viable competition. Thank you for reading.

**Comment by Doksha** [comment]
I am guessing you are suggesting each of the 4 keys is a separate input (any 3 can overlap while a 4th is still usable) Its cool in concept, but I dont really think it adds much in terms of gameplay possibilities. It would be a cool mod, but would require altering how the game registers hits (probably, maybe it still records the 4 keys as separate, I dont know)

It is definitely a cool concept, probably would best be associated with a separate input keybind settings for the mod (ie Z X M1 M2 for NM, A S D F for 4K)

**Reply by Dip9107** [reply]
Yes that’s exactly what I mean, should’ve clarified my bad. Anyways, I think it would definitely be game changing because if players can stream up to 330bpm with only two fingers at once could you imagine what a player like aetrna could do with four at his disposal at one time?

**Reply by SevenChords** [reply]
would certainly be interesting to see, but from what i can tell a rework of the inputs would have to happen if it were to be introduced as a mod as it seems that lazer just records mouse and keyboard input as the same 2 buttons that are pressed, but as a separate gamemode it would certainly be interesting

**Reply by Dip9107** [reply]
Yeah I believe that is what would have to happen and I am sure there are currently much more important things at hand for devs currently, but in the future when most of the important stuff is taken care of I feel this could be game changing if implemented correctly.
